packages/components/src/Message/Message.tsx

```diff
@@ -34,18 +33,11 @@ export const Message: React.FC<MessageProps> = ({
     >
       <Box display="flex" alignItems="center" variant="messages.title">
```

Comment:
why do we need `flex` and `align-items: center` here?

Reply:
We dont need it 👍🏼